activerecord/test/models/project.rb

```diff
@@ -2,7 +2,7 @@
 
 class Project < ActiveRecord::Base
   belongs_to :mentor
-  has_and_belongs_to_many :developers, -> { distinct.order "developers.name desc, developers.id desc" }
+  has_and_belongs_to_many :developers, -> { distinct.order "developers.id desc" }
```

Comment:
Can you use another association for the test case?
Any test case isn't failed even if the `developers.name desc` is removed, but some test cases are intended to be ordered by `developers.name desc`. ecd46231ff97f6735b62543416a230390fa22b21